text/3771-demote-i686-pc-windows-gnu.md

```diff
@@ -0,0 +1,172 @@
+- Feature Name: none
+- Start Date: (fill me in with today's date, 2025-02-11)
+- RFC PR: [rust-lang/rfcs#3771](https://github.com/rust-lang/rfcs/pull/3771)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Demote target `i686-pc-windows-gnu` from Tier 1 to Tier 2 (with host tools) to better reflect its current maintenance and usage status.
+
+# Motivation
+[motivation]: #motivation
+
+## Background
+
+Rust has supported Windows ever since before 1.0, with two different flavors of Windows targets: MSVC-based and GNU-based.
+MSVC-based targets (for example the main Windows target `x86_64-pc-windows-msvc`) use Microsoft's native `link.exe` linker and libraries, while GNU-based targets (like `i686-pc-windows-gnu`) are built entirely from free software components like `gcc`, `ld`, and MinGW.
```

Comment:
Nitpick: technically MSVC wasn't supported until, I think, 1.2. Or at least not as tier 1 or 2.